client/my-sites/customer-home/cards/actions/quick-links/style.scss

```
@@ -2,8 +2,16 @@
 @import "@wordpress/base-styles/mixins";
 
 .quick-links.foldable-card.card {
-	@include breakpoint-deprecated( ">660px" ) {
-		border-radius: 3px;
+	padding: 1rem;
+	border-radius: 3px;
+
+	@media only screen and (max-width: $break-small) {
+		border-radius: 0;
+	}
```

The border radius query is not mobile-first, it should be like this:
```suggestion
	padding: 1rem;
	border-radius: 0;

	@media only screen and (min-width: $break-small) {
		border-radius: 3px;
	}
```

---

Thanks!